charts/spring-service/templates/deployment.yaml

```diff
@@ -105,6 +105,37 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.serviceName }}-external-secret
                   key: "SPRING_DATASOURCE_PASSWORD"
+            {{- else if .Values.env.springDatasourceFromSecretWithRLS }}
+            - name: "SPRING_DATASOURCE_URL"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.serviceName }}-external-secret
+                  key: "SPRING_DATASOURCE_URL"
+            - name: "SPRING_DATASOURCE_USERNAME"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.serviceName }}-external-secret
+                  key: "SPRING_DATASOURCE_USERNAME"
+            - name: "SPRING_DATASOURCE_PASSWORD"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.serviceName }}-external-secret
+                  key: "SPRING_DATASOURCE_PASSWORD"
+            - name: "SPRING_FLYWAY_URL"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.serviceName }}-external-secret
+                  key: "SPRING_FLYWAY_URL"
```

Comment:
Is this required, when setting username and password? If it can be omitted and `datasource.url` is used instead, I would prefer that.

Reply:
Nice catch. Confirmed by the docs:
```
	/**
	 * JDBC url of the database to migrate. If not set, the primary configured data source
	 * is used.
	 */
	private String url;
```
... and testing locally confirms it works.